This patch fixes these new errors:

| XpNotifyPdm.c:234:10: error: cast from pointer to integer of different size [-Werror=pointer-to-int-cast]
| XpNotifyPdm.c:271:10: error: cast from pointer to integer of different size [-Werror=pointer-to-int-cast]
| XpNotifyPdm.c:286:10: error: cast from pointer to integer of different size [-Werror=pointer-to-int-cast]


Upstream-Status: Pending

Signed-off-by: Saul Wold <sgw@linux.intel.com>

Index: libXp-1.0.1/src/XpNotifyPdm.c
===================================================================
--- libXp-1.0.1.orig/src/XpNotifyPdm.c
+++ libXp-1.0.1/src/XpNotifyPdm.c
@@ -231,7 +231,7 @@ XpGetPdmStartParams (
 	/*
 	 * Error - cannot determine or establish a selection_display.
 	 */
-	return( (Status) NULL );
+	return( (Status) 0 );
     }
 
     /*
@@ -268,7 +268,7 @@ XpGetPdmStartParams (
 	    XCloseDisplay( *selection_display );
 	    *selection_display = (Display *) NULL;
 	}
-	return( (Status) NULL );
+	return( (Status) 0 );
     }
 
     status = XmbTextListToTextProperty( *selection_display, list, 6,
@@ -283,7 +283,7 @@ XpGetPdmStartParams (
 	    XCloseDisplay( *selection_display );
 	    *selection_display = (Display *) NULL;
 	}
-	return( (Status) NULL );
+	return( (Status) 0 );
     }
 
     *type              = text_prop.encoding;
